internal/controller/controllers/agent_controller_test.go

```diff
@@ -748,6 +748,66 @@ var _ = Describe("agent reconcile", func() {
 		Expect(agent.Status.DebugInfo.EventsURL).To(HavePrefix(expectedEventUrlPrefix))
 	})
 
+	It("validate Logs URL", func() {
+		serviceBaseURL := "http://acme.com"
```

Comment:
Super minor nit, but I usually prefer using `example.com` as it's a domain specifically reserved for being used in test and example code. ref: https://www.iana.org/domains/reserved

Reply:
Modified it, thank you!